apps/frontend/src/app/Database/DataManagers/CharacterData.ts

```diff
@@ -333,6 +333,9 @@ export class CharacterDataManager extends DataManager<
         else this.set(c.key, c)
       })
 
+      // Somnia shouldn't be removed on import.
+      idsToRemove.delete('Somnia')
```

Comment:
If I import a GO json without Somnia/QC, she still ends up sticking around

Reply:
Yeah because this just bypass the removal check. I am not sure how complicated I want to make this import logic just to accommodate Somnia.